content/SCALE/SCALETutorials/Apps/InstallWGEasyApp.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "Installing the WG-Easy Application"
+description: "Provides installation instructions for users migrating from the deprecated SCALE OpenVPN service to the new WG-Easy application." 
+weight: 68
+aliases:
+tags:
+- scaleopenvpn
+---
+
+
+{{< include file="content/_includes/SCALEServiceDeprecationNotice.md" type="page" >}}
+
+This article provides instructions on migrating from the SCALE OpenVPN client and server services to the new WG-Easy application. 
+This application is an open source utility that provides fast incremental file transfers TrueNAS SCALE uses as a replacement for the deprecated SCALE OpenVPN services. 
```

Comment:
I think here forgot to update after copy pasting? :)

Reply:
yep... gone in the next commit.